app/models/concerns/smooch_capi.rb

```diff
@@ -223,33 +223,32 @@ def capi_send_message_to_user(uid, text, extra, _force = false)
       req = Net::HTTP::Post.new(uri.request_uri, 'Content-Type' => 'application/json', 'Authorization' => "Bearer #{self.config['capi_permanent_token']}")
       req.body = payload.to_json
       response = http.request(req)
-      ret = nil
-      if response.code.to_i < 400
-        ret = response
-      else
+      if response.code.to_i >= 400
         # FIXME: Understand different errors that can be returned from Cloud API and have specific reports
         # response_body = self.safely_parse_response_body(response)
         e = Bot::Smooch::CapiMessageDeliveryError.new('Could not send message using WhatsApp Cloud API')
         CheckSentry.notify(e,
           uid: uid,
           type: payload.dig(:type),
           template_name: payload.dig(:template, :name),
-          template_language: payload.dig(:template, :language, :code)
+          template_language: payload.dig(:template, :language, :code),
+          error: response.body
```

Comment:
We don't know for CAPI yet because we're not using it on live yet... but I have left a comment on line 227 above to revisit it in the future once we know more about what can be returned:
```
FIXME: Understand different errors that can be returned from Cloud API and have specific reports
```